docs/designs/conceptual-versioning.md

```diff
@@ -366,48 +367,30 @@ _site/en-us/group-01ddf122/dotnet/api/system.string/index.html
 
 - Content for `.manifest.json`
 
-    ##### Ideal output
-
-    ```json
-    {
-        "files":[
-            {
-                "siteUrl": "/{SitePath}",
-                "outputPath": "{outputPath}",
-                "sourcePath": "{sourcePath}",
-                "monikers": [
-                    "moniker1",
-                    ...
-                ]
-            },
-        ]
-    }
-    ```
-
-    ##### Legacy output
-
-    ```json
-    {
-        "groups":[
-            "group": "{groupId}",
+```json
```

Comment:
@fenxu is working on a new publish pipeline that consumes `.publish.json`, that one uses ideal output over `.manifest.json`

Reply:
Yes, he is still writing the document about the new design, we should update this document after that finished.

---
In reply to: [279676645](https://github.com/dotnet/docfx/pull/4460#discussion_r279676645) [](ancestors = 279676645)